docs/frontend/styleguide.md

```diff
@@ -0,0 +1,70 @@
+# Shopsys Framework StyleGuide
```

Comment:
Why did you add the same file twice?

Reply:
Because of standalone styleguide without project. Should we add only link to github docs md file?

Reply:
Yes, we don't want to have duplicated files so it would be great to just add a link

Reply:
Removed.